app/models/pafs_core/area.rb

```diff
@@ -6,7 +6,8 @@ class Area < ApplicationRecord
       COUNTRY_AREA = "Country",
       EA_AREA      = "EA Area",
       PSO_AREA     = "PSO Area",
-      RMA_AREA     = "RMA"
+      RMA_AREA     = "RMA",
+      AUTHORITY_AREA = "Authority"
```

Comment:
What is an authority area as opposed to an RMA area?

Reply:
I've decided not to add new table for authority types, but add them to this existing one as all required fields are already present. So this new "Authority" area is just a flag to mark authority types